src/_tutorials/server/cmdline.md

```diff
@@ -9,19 +9,15 @@ prevpage:
   title: Get started with the Dart VM
```

Comment:
Ah yes. It all used to be under `dart-vm` until we painfully changed it to server late last year. With the memory of that long, drawn-out process, I was trying to avoid changing URLs, for now.

Here, since I was looking these up to remember just how painful this change was, are some relevant PRs & issues.

Related PRs: #1019, #1274, #1284, #1286, #1295
Related issues: #245, #965, #1281, #1285

Reply:
Yeah. I think it's fine to postpone this to a later point after I/O